addons/pointwatch/pointwatch.lua

```diff
@@ -120,7 +120,8 @@ packet_handlers = {
         accolades.current = p['Unity Points']
         ep.current = p['Current Exemplar Points']
         ep.tnml = p['Required Exemplar Points']
-        ep.master_level = p['Master Level']
+        ep.master_level = tnml[p['Required Exemplar Points']]
```

Comment:
Have you tested what happens when you've not unlocked Master Levels? Isn't this inner value `0` in that case? and if so, I'm pretty sure this would evaluate to `nil`, which I believe would maybe/probably error.

Reply:
I can try it on my mule, though I have no way to test the case where a character is 99 but doesn't have master levels unlocked, if that matters.

Reply:
Confirmed that on a character without ML unlocked it shows "ML0" as expected.

Reply:
I don't see how that's possible since there is no `0` key in your table... so... how is that possible? 🤔

Reply:
Oh, or are you saying that even without master breaker it says that Required Exemplar points is 2500?

Reply:
Oh, jk, it does have 2500 in it, so no problem at all (although, in a perfect world, would be nice if it would just, not display ML/EP at all if it's not applicable but, that's too much work ;p

Reply:
I'll add that functionality at some unspecified point in the future if I don't entirely rewrite the addon first lol